15/Dockerfile.c9s

```diff
@@ -0,0 +1,83 @@
+FROM quay.io/sclorg/s2i-core-c9s:c9s
+
+# PostgreSQL image for OpenShift.
+# Volumes:
+#  * /var/lib/pgsql/data   - Database cluster for PostgreSQL
+# Environment:
+#  * $POSTGRESQL_USER     - Database user name
+#  * $POSTGRESQL_PASSWORD - User's password
+#  * $POSTGRESQL_DATABASE - Name of the database to create
+#  * $POSTGRESQL_ADMIN_PASSWORD (Optional) - Password for the 'postgres'
+#                           PostgreSQL administrative account
+
+ENV POSTGRESQL_VERSION=15 \
+    POSTGRESQL_PREV_VERSION=13 \
+    HOME=/var/lib/pgsql \
+    PGUSER=postgres \
+    APP_DATA=/opt/app-root
+
+ENV SUMMARY="PostgreSQL is an advanced Object-Relational database management system" \
+    DESCRIPTION="PostgreSQL is an advanced Object-Relational database management system (DBMS). \
+The image contains the client and server programs that you'll need to \
+create, run, maintain and access a PostgreSQL DBMS server."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="PostgreSQL 15" \
+      io.openshift.expose-services="5432:postgresql" \
+      io.openshift.tags="database,postgresql,postgresql15,postgresql-15" \
+      io.openshift.s2i.assemble-user="26" \
+      name="sclorg/postgresql-15-c9s" \
+      com.redhat.component="postgresql-15-container" \
+      version="1" \
+      usage="podman run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 sclorg/postgresql-15-c9s" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+EXPOSE 5432
+
+COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
+
+# This image must forever use UID 26 for postgres user so our volumes are
+# safe in the future. This should *never* change, the last test is there
+# to make sure of that.
+RUN  { yum -y module enable postgresql:15 || :; } && \
```

Comment:
This is not needed as for see let's use it for 15/Dockerfile.rhel8.
`yum -y module enable postgresql:15`

Reply:
It's meant to be a generic command that should work everywhere, whether postgresql is a module or not. Otherwise we'd need to have some label in the multispec that would define whether to work with modules or not and then add a condition in the Dockerfile. I think this `|| :` is pretty nice hack, that with combination of checking the version done in #477 should be pretty stable.